turbo.json

```diff
@@ -1,13 +1,13 @@
 {
-    "$schema": "https://turbo.build/schema.json",
-    "tasks": {
-        "dev": {
-            "persistent": true,
-            "cache": false
-        },
-        "//#dev:generate-css-module-declarations": {
-            "inputs": ["**/*.module.css"],
-            "outputs": ["**/*.module.css.d.ts"]
-        }
+  "$schema": "https://turbo.build/schema.json",
+  "tasks": {
+    "dev": {
+      "persistent": true,
+      "cache": false
+    },
+    "dev:generate-css-module-declarations": {
+      "inputs": ["**/*.module.css"],
+      "outputs": ["**/*.module.css.d.ts"]
```

Comment:
Note: `turbo watch` does not cache outputs today, so you won't have outputs that you're caching until you use `turbo run` with this task. Adjust as you see fit for your workstreams!